bitnami/kong/README.md

```diff
@@ -0,0 +1,326 @@
+|----------------------------------------|--------------------------------------------------------------------------------------------------------|---------------------------------------------------------|
+| `kong.livenessProbe.enabled`                | Enable/disable the Liveness probe                                                                      | `true`                                                  |
+| `kong.livenessProbe.initialDelaySeconds`    | Delay before liveness probe is initiated                                                               | `60`                                                    |
+| `kong.livenessProbe.periodSeconds`          | How often to perform the probe                                                                         | `10`                                                    |
+| `kong.livenessProbe.timeoutSeconds`         | When the probe times out                                                                               | `5`                                                     |
+| `kong.livenessProbe.successThreshold`       | Minimum consecutive successes for the probe to be considered successful after having failed.           | `1`                                                     |
+| `kong.livenessProbe.failureThreshold`       | Minimum consecutive failures for the probe to be considered failed after having succeeded.             | `6`                                                     |
+| `kong.readinessProbe.enabled`               | Enable/disable the Readiness probe                                                                     | `true`                                                  |
+| `kong.readinessProbe.initialDelaySeconds`   | Delay before readiness probe is initiated                                                              | `5`                                                     |
+| `kong.readinessProbe.periodSeconds`         | How often to perform the probe                                                                         | `10`                                                    |
+| `kong.readinessProbe.timeoutSeconds`        | When the probe times out                                                                               | `5`                                                     |
+| `kong.readinessProbe.failureThreshold`      | Minimum consecutive failures for the probe to be considered failed after having succeeded.             | `6`                                                     |
+| `kong.readinessProbe.successThreshold`      | Minimum consecutive successes for the probe to be considered successful after having failed.           | `1`                                                     |
+| `kong.resources`                            | Configure resource requests and limits                                                                 | `nil`                                                   |
```

Comment:
The idea of dividing between "Deployment parameters" and "Kong parameters" is to put everything related to Kubernetes and the deployment definition under the 1st section, while you put everything related to configure Kong under the second. Therefore, probes and resources should be in the 1st section.
On the other hand, please use this format:
```
| `resources.limits` | The resources limits for Kong containers | `{}` |
| `resources.requests`  | The requested resources for Kong containers  | `{}`  |
| `livenessProbe` | Liveness probe configuration for Kong  | `Check values.yaml file` |
| `readinessProbe` | Readiness probe configuration for Kong | `Check values.yaml file` |
```
Please note that probes are treated as a YAML object and `kong.xxxx.enabled` dissappears

Reply:
Done